src/content/topics/integrations/app-performance/honeycomb/index.mdx

```diff
@@ -9,18 +9,19 @@ published: true
 <CalloutTitle>Honeycomb integrations are Pro and Enterprise features</CalloutTitle>
 <CalloutDescription>
 
-Honeycomb integrations are available to customers on a Pro or Enterprise plan. To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To upgrade your plan, [contact Sales](https://launchdarkly.com/contact-sales/).
+LaunchDarkly’s Pro or Enterprise plan customers can use Honeycomb integrations. To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To upgrade your plan, [contact Sales](https://launchdarkly.com/contact-sales/). You can sign up for a [free Honeycomb account] (https://bit.ly/3qoeonY).
 
 </CalloutDescription>
 </Callout>
 
+## About Honeycomb
+
+[Honeycomb] (https://www.honeycomb.io/?&utm_source=press%20release&utm_medium=pr&utm_campaign=funding&utm_content=series-c-funding-announcement) is the leading [observability] (https://www.honeycomb.io/blog/observability-5-year-retrospective/) platform used by high-performance engineering teams to quickly visualize, analyze, and improve cloud application quality and performance.  [Sign up for a free Honeycomb account here] (https://bit.ly/3qoeonY).
+
 ## Overview
 
 This topic explains the LaunchDarkly Honeycomb integrations available, including events and triggers.
```

Comment:
```suggestion
## Overview

This topic explains the LaunchDarkly Honeycomb integrations available, including events and triggers.

## About Honeycomb

[Honeycomb](https://www.honeycomb.io/) is an observability platform used by high-performance engineering teams to quickly visualize, analyze, and improve cloud application quality and performance. To sign up for a free Honeycomb account, [visit their signup page](https://ui.honeycomb.io/signup).
```

Reply:
The Overview section is at the top of every page. It's a standard element of our documentation. 

Also corrected links, removed sales-oriented language, de-obfuscated truncated link.

Reply:
What is the problem in using this link https://bit.ly/3qoeonY instead of https://ui.honeycomb.io./signup?  We want to use the shortened link as it gives the ability to do analytics on the referral traffic.  The link besides the term "Observability" is not a sales link, but a link to a blog.  What was the issue there?  If you are uncomfortable with a blog link, can you include this link https://docs.honeycomb.io/getting-started/learning-about-observability/ instead?

Reply:
Linking to bit.ly and other truncated links, rather than directly to a source URL, obfuscates the destination and can damage user trust because it prevents them from knowing where they're going when they click a link.

The "sales-oriented" language I referred to was the unnecessary adjective "leading". We don't generally link to other companies' content unless it's necessary for a user to complete a procedure or configuration step. By minimizing the frequency of external links in our docs, we prevent broken links which affect user experience and our site's SEO.

Reply:
Not a problem.  Can we use this link instead? https://ui.honeycomb.io/signup?&utm_source=launchdarkly&utm_medium=partner&utm_campaign=signup&utm_keyword=&utm_content=free-product-signup